README.md

```diff
@@ -182,22 +225,29 @@ lbaf-vt-data-files-validator --dir_path=<project-path>/data/nolb-8color-16nodes-
 # and will use them for validation process
 lbaf-vt-data-files-validator --dir_path=../../data/nolb-8color-16nodes-data --file_prefix=data --file_suffix=json
 ```
+</details>
 
-## VT data Extractor
+<details>
+<summary><b>VT Data Extractor</b></summary>
 
-VT data Extractor extracts phases from VT stats files.
+<br />
+The VT Data Extractor extracts phases from VT stats files.
```

Comment:
I wonder if we should not use small caps for `vt` everywhere now, for consistency.

Reply:
Fixed